shared/src/commonMain/kotlin/Util.kt

```diff
@@ -0,0 +1,7 @@
+import org.lighthousegames.logging.logging
+
+private val logging = logging("ChargeHub LOG")
+
+fun debug(log: Any?) {
```

Comment:
Can you give some info about the logging you added?

Reply:
This is from a logging library named lighthouse game. It allows you to log easily in both iOS and android. The string I passed in the constructor is simply the "Tag" for android debugging. It would work the same as Log.d("ChargeHub LOG", "yourDebug").

I created this function here so we didn't need to create a new instance each time when trying to debug and we could simply just call this function